documents/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ef-docs",
+  "version": "5.1.0",
```

Comment:
Do we want to include `@refinitiv-ui/*` scope into this package as well ? Something like `@refinitiv-ui/docs` would match our naming convention.

Reply:
Also, this is the first official release, should we version it as `5.0.0` ?

Reply:
It shouldn't matter because it's not npm module but your proposal name could add more beauty to the project :)

Reply:
Yep, just for the aesthetics. I'll update it.